modules/project_cleanup/function_source/main.go

```diff
@@ -224,6 +243,46 @@ func getCleanUpTagKeysOrTerminateExecution() bool {
 	return result
 }
 
+func getFeedsListFromEnv(envVariableName string) []*regexp.Regexp {
+	targetIncludedFeeds := os.Getenv(envVariableName)
+	logger.Println("Try to get Cai Feeds list")
```

Comment:
```suggestion
	logger.Println("Try to get CAI Feeds list")
```
here and every other occurence